charts/gatekeeper/README.md

```diff
@@ -86,14 +86,15 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | image.pullPolicy                             | The image pull policy                                                                  | `IfNotPresent`                                                            |
 | image.repository                             | Image repository                                                                       | `openpolicyagent/gatekeeper`                                              |
-| image.release                                | The image release tag to use                                                           | Current release version: `v3.6.0-beta.3`                                         |
+| image.release                                | The image release tag to use                                                           | Current release version: `v3.6.0-beta.3`                                  |
 | image.pullSecrets                            | Specify an array of imagePullSecrets                                                   | `[]`                                                                      |
 | resources                                    | The resource request/limits for the container image                                    | limits: 1 CPU, 512Mi, requests: 100mCPU, 256Mi                            |
 | nodeSelector                                 | The node selector to use for pod scheduling                                            | `kubernetes.io/os: linux`                                                 |
 | affinity                                     | The node affinity to use for pod scheduling                                            | `{}`                                                                      |
 | tolerations                                  | The tolerations to use for pod scheduling                                              | `[]`                                                                      |
 | controllerManager.priorityClassName          | Priority class name for controller manager                                             | `system-cluster-critical`                                                 |
-| controllerManager.exemptNamespaces           | The namespaces to exempt                                                                                   | `[]`                                                 |
+| controllerManager.exemptNamespaces           | The namespaces to exempt                                                               | `[]`                                                                      |
+| controllerManager.exemptNamespacePrefixes    | The namespace prefixes to exempt                                                       | `[]`                                                                      |
```

Comment:
Please update changes to the chart readme here: https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/README.md instead of this file.

Reply:
I see you have updated that file. please revert changes in this file as it will be updated when we cut a new release.